amf-client/shared/src/test/resources/org/raml/parser/examples/include-json-schema/output.txt.jvm

```diff
@@ -10,8 +10,8 @@ Level: Violation
 /name expected type: String, found: Integer
 
   Level: Violation
-  Target: file://user.json/#/definitions/User/example/bad
-  Property: file://user.json/#/definitions/User/example/bad
+  Target: file://amf-client/shared/src/test/resources/org/raml/parser/examples/include-json-schema/input.raml#/declarations/types/User/example/bad
```

Comment:
Before, the type User was the external json type with merged examples. Now, the external json type is in the inherits and User is a normal Raml type